test/src/lib/Index.js

```diff
@@ -950,16 +949,17 @@ describe('AutoIndex', function() {
 
     pred = pred.toDisjunctiveNormalForm();
 
-    // the results end up small enough that the first index is good enough
-    // for all subpred cases
     for ( var i = 0; i < pred.args.length; i++ ) {
       var subpred = pred.args[i];
-      idxInstance
+      var plan = idxInstance
```

Comment:
Can we get comments in the `beforeEach` and/or in the tests explaining why we expect certain `length`s and `size()`s? It's hard to review test changes like this without much context on what's being setup and why we expect what we expect.